UndertaleModLib/Models/UndertaleTexturePageItem.cs

```diff
@@ -157,7 +157,7 @@ public void ReplaceTexture(SKBitmap replaceImage, bool disposeImage = true)
             TextureWorker worker = new TextureWorker();
             SKBitmap embImage = worker.GetEmbeddedTexture(TexturePage); // Use SetPixel if needed.
 
-            SKCanvas g = new SKCanvas(embImage);
+            SKCanvas g = new(embImage);
```

Comment:
is this our coding style? personally dont like this syntax as much

Reply:
I am not sure actually, but there are a lot of this in the source code - https://github.com/search?q=repo%3Akrzys-h%2FUndertaleModTool%20new(&type=code
Also, what's wrong with it?
In my opinion, it should be either `var a = new Class()` or `Class a = new()`, why duplicate?

Reply:
And Visual Studio suggests to use `new()`
![image](https://github.com/Miepee/UndertaleModTool/assets/45438625/633b99f4-c1b9-44b4-b3c6-91d48b0f0c4b)

Reply:
I personally prefer `var foo = new Class()`. we should probably set up style guides at one point, but currently that's out of scope here.